frontend/src/pages/Login.tsx

```diff
@@ -1,7 +1,40 @@
+import React from 'react';
+import GoogleLogin from 'react-google-login';
 import './Login.css';
+import axios from 'axios';
 
 function Login() {
-    return <h1>Login</h1>;
+    const onFailure = () => {
+        console.log('Invalid account');
+    };
+
+    const onSuccess = (googleData) => {
+        axios
+            .post('http://localhost:5000/api/users/login', {
+                credential: googleData.tokenId,
+            })
+            .then(function (response) {
+                console.log(response.data);
+            });
+    };
+
+    return (
+        <div className="loginPage">
+            <header className="loginPageHeader">
+                <h1>Welcome to AEYB</h1>
+                <GoogleLogin
+                    className="loginBox"
+                    clientId="931818604859-0jd0r03np411c0v0pp89daplg1eansep.apps.googleusercontent.com"
+                    onSuccess={onSuccess}
```

Comment:
I think it would be best if this was in a config file, just to make it easier to change

Reply:
when do you think those would need to be changed?